app/templates/users/user/photos.hbs

```diff
@@ -0,0 +1,26 @@
+<EmberWormhole @to='navbar-wormhole'>
+  <PageActionsButtons @pageActions={{pageActions}} />
+</EmberWormhole>
+
+<TabbedView @tabItems={{tabItems}}>
```

Comment:
I want to verify whether we can use `{{@tabItems}}` or `{{this.tabItems}}` here. I belive tabItems originates at the user route, so I'm not sure? `this` is for controller originated properties, and `@` is for named arguments, so we'll see if route properties are just supposed to be 'bare'

Reply:
`this.tabItems` seems to work fine, so I'm also applying it for the other tab routes under `app/templates/users/user/` (i.e. `groups`, `mail`, `mandates`, etc.)

Reply:
I'm still not entirely sure why this works? or what the rules are about properties defined on routes.
I've thought about it some more, and in principle the `tabItems` and `pageActions` properties should ideally not be coming from the route, but from the controller/component js file that belongs to a template. (I think that means we are supposed to pass any relevant info from the route to the controller in order to allow the controller to provide these properties) But that's for another PR with a focus on refactors, rather than a new feature.